core/src/main/java/io/confluent/kafka/schemaregistry/leaderelector/kafka/SchemaRegistryCoordinator.java

```diff
@@ -188,6 +206,15 @@ protected Map<String, ByteBuffer> onLeaderElected(
         leaderKafkaId = kafkaMemberId;
         leaderIdentity = memberIdentity;
       }
+      if (eligible && memberIdentity.isLeader()) {
+        if (existingLeaderIdentity != null) {
+          log.warn("Multiple leaders found in group [{}, {}].",
+                  existingLeaderIdentity, memberIdentity);
+          multipleLeadersFound = true;
```

Comment:
After setting multipleLeadersFound to true, set existingLeaderKafkaId and existingLeaderIdentity to null